Git-Pull-vs-Git-Fetch.md

```diff
@@ -0,0 +1,18 @@
+<h1>Git Pull vs Git Fetch:</h1>
+
+These two commands are regularly used by git users. Let's see the difference between both commands.
+
+
+A clone is simply a copy of a repository. It is something like where you can download source code from some other repository.
```

Comment:
I think the phrasing could be improved a little here.
"_It is something like where you can download source code from some other repository_." could be changed to something similar to this:
"It is similar to downloading source code from a different source that is not version-controlled."